platform_versioned_docs/version-24.1/supported_software/fusion/fusion.mdx

```diff
@@ -3,70 +3,31 @@ title: "Fusion v2 file system"
 description: "Fusion file system"
 ---
 
-Seqera Platform 22.4 adds official support for the Fusion v2 file system.
+Fusion v2 is a lightweight container-based client that enables containerized tasks to access data in Amazon S3, Google Cloud, or Azure Blob Storage buckets using POSIX file access semantics. Depending on your data handling requirements, Fusion can improve pipeline throughput and reduce cloud computing costs. 
 
-Fusion v2 is a lightweight container-based client that enables containerized tasks to access data in Amazon S3, Google Cloud, or Azure Blob Storage buckets using POSIX file access semantics. Depending on your data handling requirements, Fusion can improve pipeline throughput, which should reduce cloud computing costs. See [here](https://www.nextflow.io/docs/latest/fusion.html#fusion-file-system) for more information on Fusion's features.
+See [here](https://www.nextflow.io/docs/latest/fusion.html#fusion-file-system) for more information on Fusion's features.
 
-Fusion relies on the Wave containers service. When Fusion v2 and Wave containers is enabled on a Seqera compute environment, the corresponding configuration values are added to your Nextflow config file when running pipelines with that compute environment:
+### Fusion mechanics
 
-```yaml
-wave {
-enabled = true
-}
+The Fusion file system implements a lazy download and upload algorithm that runs in the background to transfer files in parallel to and from object storage into a container-local temporary folder. This means that the performance of the disk volume used to carry out your computation is key to achieving maximum performance.
 
-fusion {
-enabled = true
-}
-```
+By default, Fusion uses the container `/tmp` directory as a temporary cache, so the size of the volume can be much lower than the actual needs of your pipeline processes. Fusion has a built-in garbage collector that constantly monitors remaining disk space on the temporary folder and immediately evicts old cached entries when necessary. 
```

Comment:
@jordeu per my [post](https://seqera.slack.com/archives/C03NRLDML1X/p1723813014022469) in product-fusion, can we include more information on the garbage collection mechanism here to contextualize our local storage recommendations?

Reply:
Forwarding my comments here:

Not sure if it's good to document this (we are going to change a bit in Fusion 2.5). Currently the default strategy is when the local disk used to store the cache its available space goes below 20% of total capacity Fusion starts cleaning the disk, and if it goes below 10% of capacity it keeps cleaning the disk and also stops any background download and blocks more often the process to slow down.

On practice what happens is that when the disk reaches 20% of available space then there is a spike of all Fusions running on that host simultaneously cleaning the disk and easily getting back to >50% because Fusion cleans at least 25% of the space that is using when <20% is reached.